imdb/views.py

```diff
@@ -0,0 +1,56 @@
+import json
+import pdb
+import sys
+import traceback
+
+import requests
+
+import SearchResults
+
+
+from bs4 import BeautifulSoup
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.core.paginator import Paginator, PageNotAnInteger, EmptyPage
+
+
+
+def search_movie(request):
+
+    movie = request.GET.get('q','')
```

Comment:
movie = request.GET.get('q','')

this line here denotes that when  a url is called the url after the / is passed to the request  and by using the GETfunction we extract any string that comes after q

and ' ' is used in case no particular string is matched so we pass an empty string to the movie 

and if the string is matched it is passed to the movie